integrations/newrelic/newrelic_integration/core/entities.py

```diff
@@ -0,0 +1,138 @@
+from typing import Optional
+
+from loguru import logger
+
+from newrelic_integration.core.paging import send_paginated_graph_api_request
+from newrelic_integration.core.utils import send_graph_api_request
+from newrelic_integration.utils import (
+    get_port_resource_configuration_by_port_kind,
+    render_query,
+)
+
+
+class EntitiesHandler:
+    async def get_entity(self, entity_guid: str) -> dict:
+        query_template = """
```

Comment:
Can you store all those templates in one file and instead of jinja use python string.fromat()?